openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -484,6 +484,10 @@ def post(self, request):
                 address already exists
             HttpResponse: 403 operation not allowed
         """
+        if settings.FEATURES.get('ENABLE_THIRD_PARTY_ONLY_AUTH') and not pipeline.running(request):
+            # if request is not running a third-party auth pipeline
+            return HttpResponseForbidden("Not allowed to register")
```

Comment:
Can we expand on the error message to add more clarity?  It can be the same error message as the string suggested above.